engine/src/main/java/org/terasology/particles/functions/generators/TextureOffsetGeneratorFunction.java

```diff
@@ -35,9 +35,9 @@ public void onEmission(TextureOffsetGeneratorComponent component, ParticleData p
         if (component.validOffsets.size() == 0) {
             return;
         }
-        
+
         final int randomOffsetIndex = random.nextInt(component.validOffsets.size());
         final Vector2f randomOffset = component.validOffsets.get(randomOffsetIndex);
-        particleData.textureOffset.set(randomOffset.getX(), randomOffset.getY());
+        particleData.textureOffset.set(randomOffset.x(), randomOffset.y());
```

Comment:
Could be simplified:
```suggestion
        particleData.textureOffset.set(randomOffset);
```